src/main/java/io/aiven/kafka/auth/json/AclPermissionType.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2023 Aiven Oy https://aiven.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.auth.json;
+
+public enum AclPermissionType {
+    ALLOW((byte) 1),
+    DENY((byte) 0);
+
+    private final byte code;
+
+    AclPermissionType(final byte code) {
+        this.code = code;
+    }
+
+    public org.apache.kafka.common.acl.AclPermissionType toNativeType() {
+        if (this.code == (byte) 0) {
+            return org.apache.kafka.common.acl.AclPermissionType.DENY;
+        } else if (this.code == (byte) 1) {
+            return org.apache.kafka.common.acl.AclPermissionType.ALLOW;
+        } else {
+            throw new UnsupportedOperationException(String.valueOf(this.code));
+        }
+    }
```

Comment:
This also could be replaced with something like
```java
public enum AclPermissionType {
    ALLOW(org.apache.kafka.common.acl.AclPermissionType.ALLOW),
    DENY(org.apache.kafka.common.acl.AclPermissionType.DENY);

    public final org.apache.kafka.common.acl.AclPermissionType nativeType;

    AclPermissionType(final org.apache.kafka.common.acl.AclPermissionType nativeType) {
        this.nativeType = nativeType;
    }
}
```
Just expressing ideas

Reply:
I went with this approach